src/FFMpeg/Filters/Audio/ImageVideoFilter.php

```diff
@@ -0,0 +1,74 @@
+<?php
+
+/*
+ * This file is part of PHP-FFmpeg.
+ *
+ * (c) Alchemy <info@alchemy.fr>
+ *
+ * For the full copyright and license information, please view the LICENSE
+ * file that was distributed with this source code.
+ */
+
+namespace FFMpeg\Filters\Audio;
+
+use FFMpeg\Filters\TPriorityFilter;
+use FFMpeg\Format\AudioInterface;
+use FFMpeg\Media\Audio;
+
+/**
+ * Converts the audio to video and `artwork` as video content.
+ *
+ * @author     pralhad.shrestha05@gmail.com
+ * @copyright  Jens Hausdorf 2018
+ * @license    MIT License
+ * @package    FFMpeg\Filters
+ * @subpackage Audio
+ */
+class ImageVideoFilter implements AudioFilterInterface
+{
+    use TPriorityFilter;
+
+    /**
+     * @var string
+     */
+    private $artwork;
+    /**
+     * @var string
+     */
+    private $preset;
+    /**
+     * @var int
+     */
+    private $priority;
+    /**
+     * @var array
+     */
+    private $extraFlags;
+
+    public function __construct(string $artwork, string $preset = null, array $flags = null, int $priority = 10)
+    {
+        $this->artwork = $artwork;
+        $this->extraFlags = $flags;
+        $this->preset = $preset;
+        $this->priority = $priority;
+    }
+
+    /**
+     * {@inheritDoc}
+     */
+    public function apply(Audio $audio, AudioInterface $format): array
+    {
+        $commands = array('-loop', 1, '-i', $this->artwork);
```

Comment:
use `['-loop', '`

Reply:
as described in #412 we'll drop support for anything lower than 7.1. Please note that the first version of the `v1.x` branch is scheduled to be released on July, 12.